pennylane/operation.py

```diff
@@ -2971,6 +2971,11 @@ def enable_new_opmath():
     """
     global __use_new_opmath
     __use_new_opmath = True
+    warnings.warn(
```

Comment:
Just making a note that there will be two warnings that pop up when used in a context manager. Not a huge concern but hey 🤷

Reply:
Oh good point 🤔 This is not ideal

Reply:
We discussed today and wouldn't worry about it too much :)

Reply:
I removed the warnings when using the context manager